apps/dashboard/app/javascript/packs/batch_connect.js

```diff
@@ -191,46 +191,48 @@ function addHideHandler(optionId, option, key,  configValue) {
  *        data-max-num-cores-for-cluster-annie-oakley: 42
  *      ]
  */
-function addMinMaxForHandler(optionId, option, key,  configValue) {
-  optionId = String(optionId || '');
+function addMinMaxForHandler(subjectId, option, key,  configValue) {
+  subjectId = String(subjectId || '');
   configValue = parseInt(configValue);
 
   const configObj = parseMinMaxFor(key);
-  const id = configObj['subjectId'];
+  const objectId = configObj['subjectId'];
   // this is the id of the target object we're setting the min/max for.
   // if it's undefined - there's nothing to do, it was likely configured wrong.
-  if(id === undefined) return;
+  if(objectId === undefined) return;
 
   const secondDimId = configObj['predicateId'];
   const secondDimValue = configObj['predicateValue'];
 
-  if(minMaxLookup[id] === undefined) minMaxLookup[id] = new Table(optionId, secondDimId);
-  const table = minMaxLookup[id];
+  // several subjects can try to change the object, so the table lookup key has to have both
+  const lookupKey = `${subjectId}_${objectId}`;
+  if(minMaxLookup[lookupKey] === undefined) minMaxLookup[lookupKey] = new Table(subjectId, secondDimId);
+  const table = minMaxLookup[lookupKey];
```

Comment:
This is the bit that's functionally different. Changing the key lookup key here to include both the subject and the object (as opposed to just being the object).